rules/upgrade-your-laptop/rule.md

```diff
@@ -0,0 +1,27 @@
+---
+type: rule
+title: Do you know when to upgrade your laptop?
+uri: upgrade-your-laptop
+authors:
+  - title: Ash Anil
+    url: https://www.ssw.com.au/people/ash
+created: 2022-09-27T05:52:37.565Z
+guid: 52b3f2c6-fcb8-42f6-b469-f1621f839b49
+---
+Electronic gadgets should have an expiry date because their performance fades as we use them frequently.
+
+For example, laptops are the IT employee's best friend which helps to code and make life easier. We spend thousands of $ and get a laptop with high specs and think it would be good for the next 10 years or so. 
```

Comment:
```suggestion
For example, in the IT industry, laptops are the tools of the trade. Employees often spend thousands of $ and get a laptop with high specs thinking it will be good for the next 10 years or so.
```

Reply:
Done